ldms/src/contrib/sampler/variorum_ldms/Makefile.am

```diff
@@ -0,0 +1,16 @@
+pkglib_LTLIBRARIES =
+lib_LTLIBRARIES =
+dist_man7_MANS =
+dist_man1_MANS =
+
+AM_CPPFLAGS = @OVIS_INCLUDE_ABS@
+AM_LDFLAGS = @OVIS_LIB_ABS@
+COMMON_LIBADD = -lsampler_base -lldms -lovis_util -lcoll \
+		@LDFLAGS_GETTIME@
+
+if ENABLE_VARIORUM_LDMS
+libvariorum_ldms_la_SOURCES = variorum_ldms.c
+libvariorum_ldms_la_LIBADD = $(COMMON_LIBADD) -lvariorum -ljansson
+pkglib_LTLIBRARIES += libvariorum_ldms.la
+dist_man7_MANS += Plugin_variorum_ldms.man
+endif
```

Comment:
I don't see the README.md mentioned here. At the very lease is should be in EXTRA_DIST.